src/mpid/ch4/src/ch4r_win.h

```diff
@@ -308,6 +308,17 @@ static inline int MPIDI_CH4R_mpi_win_set_info(MPIR_Win * win, MPIR_Info * info)
     if (mpi_errno != MPI_SUCCESS)
         MPIR_ERR_POP(mpi_errno);
 
+    /* netmod or shmmod can update its internal setting based on user updated info. */
+    mpi_errno = MPIDI_NM_mpi_win_set_info_hook(win);
```

Comment:
Is this `win_set_info_hook` invocation required for `MPI_Win_create` and `MPI_WIN_allocate`?

Reply:
I think netmod can do info updates at `win_{create|allocate}_hook`. If the internal info updates are the same, the netmod can have a common internal routine for it.